ignite-generator/src/app/index.es

```diff
@@ -428,8 +428,22 @@ export class AppGenerator extends Generators.Base {
 
     const done = this.async()
     const dir = `${Shell.pwd()}/${this.name}`
-    // run the npm command
-    this.spawnCommand('npm', ['install'], {cwd: dir, stdio: 'ignore'})
+    let command = ''
+    let commandOpts = []
+
+    // Use Yarn if it is installed
+    if (!Shell.which('yarn')) {
```

Comment:
in the future I always found `if (trueCondition) true else false condition` more readable than `if (not true) false else true condition`

I know that's a nit, but if Ruby programming taught me anything, it's that removing inversions like that increases readability speed more than I ever thought it would.